Formula/newrelic.rb

```diff
@@ -0,0 +1,29 @@
+class Newrelic < Formula
+  desc "The New Relic Command-line Interface"
+  homepage "https://github.com/newrelic/newrelic-cli"
+  version = "0.2.3"
+  url "https://github.com/newrelic/newrelic-cli/archive/v#{version}.tar.gz"
```

Comment:
`version` is a method, and testing shows that leaving the version in the URL it does properly detect.  So instead of setting it, we should revert back to the auto-detection and the lower L14 change still works as expected.